examples/controls/src/app.js

```diff
@@ -33,8 +33,8 @@ export default class App extends Component {
         zoom: 3.5,
         bearing: 0,
         pitch: 0,
-        width: window.innerWidth,
-        height: window.innerHeight,
+        width: 500,
```

Comment:
No particular reason really... Some examples were using window size, some were using default props, and I just wanted to align them.